### PR TITLE
[feat] Merchant customers sad path v2

### PR DESCRIPTION
**Pull request by MiTOBrien**
Updated controller and spec file to use rescue_from and Error Serializer - Confirmed tests pass.

### CODE REVIEW

File: app/controllers/api/v1/merchant_customers_controller.rb
Comment (Crosswolfv1) [inline app/controllers/api/v1/merchant_customers_controller.rb:1]: do we want to leave the commented code in there?

File: spec/requests/api/v1/merchant_customer_spec.rb
Comment (Crosswolfv1) [inline spec/requests/api/v1/merchant_customer_spec.rb:1]: can you remove the binding.pry?

**Review by Crosswolfv1 — CHANGES_REQUESTED** [review]
please review comments

File: app/controllers/api/v1/merchant_customers_controller.rb
Comment (MiTOBrien) [reply app/controllers/api/v1/merchant_customers_controller.rb:1]: Commented code has been removed.

File: spec/requests/api/v1/merchant_customer_spec.rb
Comment (MiTOBrien) [reply spec/requests/api/v1/merchant_customer_spec.rb:1]: Binding.pry has been removed.

**Review by Crosswolfv1 — APPROVED** [review]
thnk you

**Pull request merged**